templates/chat/activity-card.hbs

```diff
@@ -18,7 +18,9 @@
     {{#if buttons}}
     <div class="card-buttons">
         {{#each buttons}}
-        <button type="button" name="{{ @key }}" {{ dnd5e-dataset dataset }}>{{{ label }}}</button>
+        <button type="button" name="{{ @key }}" {{ dnd5e-dataset dataset }}>
+            {{{ icon }}} {{{ label }}}
```

Comment:
Might have trouble styling this if we don't have more predictable structure here. The icon is maybe ok if we assume it's always an `<i>` or `<dnd5e-icon>` but the label should probably be wrapped in a `<span>` either here or in `_usageChatButtons`.